webpack.config.js

```diff
@@ -173,21 +174,12 @@ const webpackConfig = {
 			getAliasesForExtensions()
 		),
 	},
-	node: {
-		console: false,
-		process: true,
-		global: true,
-		Buffer: true,
-		__filename: 'mock',
-		__dirname: 'mock',
-		fs: 'empty',
-		crypto: false,
-		stream: false,
-	},
+	node: false,
 	plugins: _.compact( [
 		new webpack.DefinePlugin( {
 			'process.env.NODE_ENV': JSON.stringify( bundleEnv ),
 			PROJECT_NAME: JSON.stringify( config( 'project' ) ),
+			global: {},
```

Comment:
hmmm... we have some places like `lib/viewport/index.js` that use `global` to grab things in the browser...

`ag global\\\.` to find 'em

Reply:
Do you know of any reason we are using `global` in some places? For example, in `client/blocks/reader-featured-video/index.jsx` there is:

```js
componentDidMount() {
   if ( this.props.allowPlaying ) {
      global.window && global.window.addEventListener( 'resize', this.throttledUpdateVideoSize );
   }
}
```

as far as I can tell, we can just drop `global` in situations like this